users/src/main/java/io/crate/operation/auth/TrustAuthentication.java

```diff
@@ -29,12 +30,16 @@
 public class TrustAuthentication implements AuthenticationMethod {
 
     static final String NAME = "trust";
-    private static final String SUPERUSER = "crate";
+    private final UserManager userManager;
+
+    public TrustAuthentication(UserManager userManager) {
```

Comment:
but the supplier can't take any arguments. In my case i need to pass the username.